DateCalculator/js/pages/timer.js

```diff
@@ -0,0 +1,30 @@
+import { Howl } from './howler.js';
```

Comment:
Лучше переложить файлик с библиотекой в папочку для библиотек)
```suggestion
import { Howl } from '../libs/howler.js';
```